### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by vitoriaHeliane**
#### What is the purpose of this pull request?

As the title says 
#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,99 +1,88 @@
...
+| `image`                     | `String`      | Link for the image of the banner                                   |
+| `mobileImage`               | `String`      | Link for the mobile image of the banner                            |
+| `description`               | `String`      | The image's description                                            |
+| `url`                       | `String`      | The URL where the image is pointing to, in case of external route  |
+| `page`                      | `String`      | The page where the image is pointing to                            |
+| `params`                    | `String`      | Parameters of the URL                                              |
+| `externalRoute`             | `Boolean`     | Indicates if the route is external or not                          |
+
+### Styles API
+:construction: :construction: :construction:
```
Comment (theomoura) [inline README.md]: Update the styles section

Comment (vitoriaHeliane) [reply README.md]: It is construction because we are waiting for the Slider API.

File: README.md [hunk 2]
```diff
@@ -1,99 +1,88 @@
-# Carousel
+# VTEX Carousel
+
+## Description
+The VTEX Carousel is an app that shows a collection of banners and it is used by the Dreamstore product.
```
Comment (GustavoMA) [inline README.md]: See changes in minicart master. You need to update from the changes that @brunojdo did yesterday.

File: README.md (hunk 1 above)
Comment (theomoura) [reply README.md]: You can add the initial content of the section, like on minicart app

File: README.md [hunk 3]
```diff
@@ -1,99 +1,134 @@
...
+
+## Description
+
+The VTEX Carousel app is a store component that shows a collection of banners, and this app is used by store theme.
+
+:loudspeaker: **Disclaimer:** Don't fork this project; use, contribute, or open issue with your feature request.
 
-Carousel is a canonical component that any VTEX store can install.
 ## Release schedule
 | Release  | Status              | Initial Release | Maintenance LTS Start | End-of-life | Dreamstore Compatibility
```
Comment (GustavoMA) [inline README.md]:
```suggestion
| Release  | Status              | Initial Release | Maintenance LTS Start | End-of-life | Store Compatibility
```

File: README.md [hunk 4]
```diff
@@ -1,99 +1,134 @@
...
-            params: {
-              type: 'string',
-              title: 'Params',
-            },
-          },
-        },
-      },
+This app uses our store builder with the blocks architecture. To know more about Store Builder [click here.](https://help.vtex.com/en/tutorial/understanding-storebuilder-and-stylesbuilder#structuring-and-configuring-our-store-with-object-object)
+
+We add the carousel as a block in our [Store](https://github.com/vtex-apps/store/blob/2462b6506cb2af86ba2e0931e08dca4783e66cfb/store/interfaces.json).
```
Comment (GustavoMA) [inline README.md]: Use master branch link.

Comment (brunojdo) [inline README.md]: Why you point the link to a commit hash?

File: README.md [hunk 5]
```diff
@@ -1,99 +1,134 @@
...
+### Blocks API
+
+When implementing this app as a block, various inner blocks may be available. The following interface lists the available blocks within carousel and describes if they are required or optional.
+
+```json
+  {
+  "carousel": {
+    "component": "Carousel"
+  }
+}
```
Comment (GustavoMA) [inline README.md]:
``` 
"carousel": {
    "component": "Carousel"
}
```

File: README.md [hunk 6]
```diff
@@ -1,99 +1,134 @@
...
+| `showDots`                  | `Boolean`      | Shows the carousel dots                                            | true |
+| `showArrows`                | `Boolean`      | Shows the carousel arrows                                          | true |
+| `height`                    | `Number`       | Set banners height                                                 | 420 |
+| `banners`                   | `Array(Banner)`| Array of banners the will be shown in the carousel                 | [] |
+
+Banner:
+
+| Prop name          | Type       | Description                                                                 | Default value |
+| ------------------ | ---------- | --------------------------------------------------------------------------- | --- |
+| `image`                     | `String`      | Link for the image of the banner                                   | `undefined` |
```
Comment (brunojdo) [inline README.md]: Instead undefined, add N/A

File: README.md [hunk 7]
```diff
@@ -1,99 +1,134 @@
...
+| `mobileImage`               | `String`      | Link for the mobile image of the banner                            | `undefined` |
+| `description`               | `String`      | The image's description                                            | `undefined` |
+| `url`                       | `String`      | The URL where the image is pointing to, in case of external route  | `undefined` |
+| `page`                      | `String`      | The page where the image is pointing to                            | `undefined` |
+| `params`                    | `String`      | Parameters of the URL                                              | `undefined` |
+| `externalRoute`             | `Boolean`     | Indicates if the route is external or not                          | false |
+| `height`                    | `Number`       | Set banners height                                                 | 420 |
```
Comment (brunojdo) [inline README.md]: Height is already defined in the carousel.

File: README.md [hunk 8]
```diff
@@ -1,99 +1,134 @@
...
+When implementing this app as a block, various inner blocks may be available. The following interface lists the available blocks within carousel and describes if they are required or optional.
+
+```json
+  {
+  "carousel": {
+    "component": "Carousel"
+  }
+}
 ```
+The carousel has no required or allowed block. So, any carousel block implementation do not need any block inside of carousel.
```
Comment (GustavoMA) [inline README.md]: `For now this block does not have any required or optional blocks.` :hammer:

File: README.md [hunk 9]
```diff
@@ -1,99 +1,134 @@
...
+| `autoplay`                  | `Boolean`      | Enable automatic banner transition                                 | true |
+| `autoplaySpeed`             | `Number`       | Set the automatic banner transition interval                       | 5 |
+| `showDots`                  | `Boolean`      | Shows the carousel dots                                            | true |
+| `showArrows`                | `Boolean`      | Shows the carousel arrows                                          | true |
+| `height`                    | `Number`       | Set banners height                                                 | 420 |
+| `banners`                   | `Array(Banner)`| Array of banners the will be shown in the carousel                 | [] |
+
+Banner:
+
+| Prop name          | Type       | Description                                                                 | Default value |
```
Comment (GustavoMA) [inline README.md]: There's no need to show default in defined objects.

File: README.md (hunk 6 above)
Comment (GustavoMA) [reply README.md]: We are using `-`, @brunojdo

File: README.md (hunk 5 above)
Comment (vitoriaHeliane) [reply README.md]: ?

**Pull request merged**